release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -1266,6 +1266,10 @@ If you are using the option `--keep-manifest-list=true`, the only valid value fo
 
 * Previously, the API server sometimes failed to create a resource and returned a 409 conflict response status code due to a conflict while updating a resource quota resource. This issue has been fixed. Now, if it receives a 409 status code, the OpenShift web console retries the request up to three times. If it continues to receive the 409 status code, the console displays an error message. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1928228[*BZ#1928228*])
 
+* Previously, when the user navigated to the *Topology* view, an error occurred if there were no templates on `openshift` namespace due to which a blank page was displayed. This issue has been fixed by handling the edge case with respect to the templates not being present in the `openshift` namespace. The user can now navigate to the *Topology* view and it loads as expected. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1952293[*BZ#1952293*])
```

Comment:
This first sentence is a little long, and specifically the last part, took me a couple times to read before I understood the meaning. I suggest breaking the sentence into two to make reading it quicker and easier. So something like: 

Previously, when the user navigated to the *Topology* view, an error occurred if there were no templates on the `openshift` namespace. If this error occurred, a blank page would display. 

Note, I added the article `the` before `openshift namespaces`. I assume this article is needed, but if it's `openshift namespace` typically doesn't need an article, it's find to leave it as is :)

Reply:
Thanks! Fixed :rocket: